### PR TITLE
PERF-5245 Create multiplanner performance workload where all plans are bad

### PR DESCRIPTION
**Pull request by dhsegel**

### CODE REVIEW

File: src/workloads/query/multiplanner/NoGoodPlan.yml [hunk 1]
```diff
@@ -0,0 +1,403 @@
...
+  Threads: 1
+  Database: *db
+  Phases:
+    OnlyActiveInPhases:
+      Active: [2]
+      NopInPhasesUpTo: *maxPhase
+      PhaseConfig:
+        Repeat: 1
+
+- Name: SimpleMultiplannerWith64Indexes
```
Comment (Fefer-Ivan) [inline src/workloads/query/multiplanner/NoGoodPlan.yml:271]: Simple?

File: src/workloads/query/multiplanner/NoGoodPlan.yml [hunk 2]
```diff
@@ -0,0 +1,403 @@
...
+    Key: QueryTemplate
+    Parameters:
+      active: [3]
+      nopInPhasesUpTo: *maxPhase
+      repeat: *queryRepeats
+      database: *db
+      collection: *coll
+      query: &query {
+        Filter: {
+          x1: {$lt: *selectivity},
```
Comment (Fefer-Ivan) [inline src/workloads/query/multiplanner/NoGoodPlan.yml:283]: So this looks like a good plan?

**Review by Fefer-Ivan — CHANGES_REQUESTED** [review]

File: src/workloads/query/multiplanner/NoGoodPlan.yml (hunk 1 above)
Comment (dhsegel) [reply src/workloads/query/multiplanner/NoGoodPlan.yml:271]: Removing.

File: src/workloads/query/multiplanner/NoGoodPlan.yml (hunk 2 above)
Comment (dhsegel) [reply src/workloads/query/multiplanner/NoGoodPlan.yml:283]: What do you mean? Wouldn't marking `index1` as hidden mean that there is no good plan?

Comment (Fefer-Ivan) [reply src/workloads/query/multiplanner/NoGoodPlan.yml:287]: Oh, now I see.
That is too easy to miss, given copy-paste nature of this workloads.
Why do we go this way?
Let's just add new "unindexed" field, for example?

Comment (Fefer-Ivan) [reply src/workloads/query/multiplanner/NoGoodPlan.yml:287]: This will be more readable.

File: src/workloads/query/multiplanner/NoGoodPlan.yml [hunk 3]
```diff
@@ -16,14 +16,14 @@ GlobalDefaults:
   # Collection name used for queries.
   coll: &coll Collection0
 
-  docCount: &docCount 1e5
+  docCount: &docCount 1e2
```
Comment (Fefer-Ivan) [inline src/workloads/query/multiplanner/NoGoodPlan.yml:19]: Should this be reverted before merge?

**Review by Fefer-Ivan — APPROVED** [review]
LGTM with a comment

**Comment by dhsegel** [comment]
evergreen merge

**Pull request merged**